audit/projects/k8s-release/iam.json

```diff
@@ -50,6 +50,12 @@
       ],
       "role": "roles/editor"
     },
+    {
+      "members": [
+        "serviceAccount:cloud-ingest-dcp@cloud-ingest-prod.iam.gserviceaccount.com"
```

Comment:
Near as I can tell this is a static service account for "Transfer Appliance" which... this isn't transferring to or from on-prem storage, so what gives?

I'm inclined to revoke this

Reply:
```
gcloud projects remove-iam-policy-binding k8s-release --member=serviceAccount:cloud-ingest-dcp@cloud-ingest-prod.iam.gserviceaccount.com --role=roles/pubsub.editor
```